app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -633,22 +633,31 @@ public void onPos(final SourceContent sourceContent, boolean b) {
                     Realm realm = Realm.getDefaultInstance();
                     WebLink link = realm.createObject(WebLink.class);
 
-                    linkPreviewViewHolder.previewLayout.setVisibility(View.VISIBLE);
-                    linkPreviewViewHolder.previewImageView.setVisibility(View.VISIBLE);
-                    linkPreviewViewHolder.descriptionTextView.setVisibility(View.VISIBLE);
-                    linkPreviewViewHolder.titleTextView.setVisibility(View.VISIBLE);
-                    linkPreviewViewHolder.titleTextView.setText(sourceContent.getTitle());
-                    linkPreviewViewHolder.descriptionTextView.setText(sourceContent.getDescription());
+                    if(sourceContent != null && (!sourceContent.getDescription().isEmpty())) {
```

Comment:
Move if(sourceContent != null) from here and add it to top and add all this code under that.